### PR TITLE
Cherry-Pick "Shrimp Mob for Metempsychosis (#862)"

### PR DESCRIPTION
**Pull request by Mnemotechnician**
# Description
Cherry-picks https://github.com/deltaV-Station/delta-v/pull/862 on behalf of the author.

This was not tested; need someone to check if it actually works before merging.

# Changelog
:cl:
- add: A shrimp morphotype was added to the failure pool of the metempsychotic machine.

### CODE REVIEW

**Comment by leonardo-dabepis** [comment]
ah yes the one I coded while high

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+- type: entity
+  name: space shrimp
+  parent: [ SimpleMobBase, FlyingMobBase, MobCombat ]
+  id: MobSpaceShrimp
+  description: Shrimply cursed...
+  suffix: METEM MACHINE ONLY #for now, only to be used as a very rare result for the metempsychosis machine. If fishing ever gets added, this suffix can be removed.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml:6]:
```suggestion
```

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        Base: shrimp
+      Dead:
+        Base: dead
+  - type: Butcherable
+    spawned:
+    - id: FoodMeatCrab
+      amount: 3
+  - type: Bloodstream
+    bloodMaxVolume: 100
+    bloodReagent: BbqSauce #australia reference
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml:43]:
```suggestion
    bloodReagent: BbqSauce # Australia reference
```

**Review by DEATHB4DEFEAT — APPROVED** [review]
Shrimply incredible.

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    speechSounds: Slime
+  - type: Puller
+    needsHands: false
+  - type: MindContainer
+    showExamineInfo: true
+  - type: Psionic
+    removable: false
+  - type: InnatePsionicPowers
+    powersToAdd:
+    - PyrokinesisPower
```
Comment (VMSolidus) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/fun.yml:60]:
```suggestion
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**